Sources/Keychain/storage/KeyStoring.swift

```diff
@@ -19,9 +19,9 @@ public final class KeyStorage: KeyStoring {
     }
 
     @discardableResult
-    public func set(data: Data, key: String) throws -> OSStatus {
+    public func set(data: Data, key: String) -> OSStatus {
         // 1. Delete any existing key before saving it
-        try delete(key: key)
+        delete(key: key)
```

Comment:
And then if this throws an error, we catch that and show a useful error to the user? We should direct them to contact support - perhaps with a link to the telegram channel or us

Reply:
https://github.com/dawnwallet/dawn-key-management/pull/7/commits/3b99223e940ad89ccc403ef0d89da93ee009d97a